### ISSUE TITLE
Issue when using loadtxt with string data in Python 3

### ISSUE THREAD

**Issue by astrofrog** [numpy/numpy#5530]
Let's say I use a data file `data.txt` containing:

```
a b c
d e f
```

If I do (in Python 3.4):

```
import numpy as np
data = np.loadtxt('data.txt', dtype='U')
print(data)
```

I get:

```
[["b'a'" "b'b'" "b'c'"]
 ["b'd'" "b'e'" "b'f'"]]
```

Note that the `b` is actually part of the string, so there is something wrong here. The same happens if I use `dtype='U'`. I am using Numpy dev as of a02f5c8fd384b5aeffd1942747b4b6c4efdcd3a4.

**Comment by juliantaylor** [comment]
this is unfortunately a known problem of the numpys text loading functions converting everything to bytes for no good reason. its everywhere and needs substantial work to fix, but its still on the agenda to fix for 1.10.

**Comment by mdickinson** [comment]
Seems to be a duplicate of #2715. It also seems to be fixed in NumPy 1.13.3.

**Comment by seberg** [comment]
Indeed, seems fixed.  We still have the annoying `encoding="bytes"` default, but this particular issue is fine.
